canonicalwebteam/docs/blueprint.py

```diff
@@ -0,0 +1,118 @@
+from urllib.parse import urlparse, urlunparse, unquote
+
+import flask
+from requests.exceptions import HTTPError
+from canonicalwebteam.yaml_responses.flask_helpers import (
+    prepare_deleted,
+    prepare_redirects,
+)
+
+from canonicalwebteam.docs.models import (
+    DiscourseDocs,
+    NavigationParseError,
+    RedirectFoundError,
+)
+
+
+def build_blueprint(url_prefix, discourse_url, frontpage_id, category_id):
+
+    discourse_blueprint = flask.Blueprint(
+        "discourse",
+        __name__,
+        template_folder="/templates",
```

Comment:
You mentioned you're going to allow the application to set the location of the templates - a good idea I think.

But I realised that actually in this module there's only one meaningful template used - `"docs/document.html"`. So the `build_blueprint` function should probably explicitly ask for just this one template path, or something. Is that possible?

Reply:
done